Test failure ClientRegressionWithRealNetworkTest

Stacktrace
```
java.lang.AssertionError: Values should be different. Actual: 0
	at com.hazelcast.client.ClientRegressionWithRealNetworkTest.testListenersAfterClientDisconnected(ClientRegressionWithRealNetworkTest.java:219)
	at com.hazelcast.client.ClientRegressionWithRealNetworkTest.testListenersAfterClientDisconnected_memberHostname_clientHostname(ClientRegressionWithRealNetworkTest.java:174)
```

http://jenkins.hazelcast.com/job/Hazelcast-pr-builder/4715/testReport/junit/com.hazelcast.client/ClientRegressionWithRealNetworkTest/testListenersAfterClientDisconnected_memberHostname_clientHostname/

--- a/hazelcast/src/test/java/com/hazelcast/client/ClientRegressionWithRealNetworkTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/client/ClientRegressionWithRealNetworkTest.java
@@ -195,9 +195,9 @@ public class ClientRegressionWithRealNetworkTest extends ClientTestSupport {
         Hazelcast.newHazelcastInstance(config);
 
         assertTrueEventually(() -> {
+            map.remove(1);
             map.put(1, 2);
             assertNotEquals(0, eventCount.get());
-            map.remove(1);
         });
     }
 

--- a/hazelcast/src/test/java/com/hazelcast/query/impl/AbstractIndexConcurrencyTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/query/impl/AbstractIndexConcurrencyTest.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.

--- a/hazelcast/src/test/java/com/hazelcast/query/impl/IndexConcurrencyTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/query/impl/IndexConcurrencyTest.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.

